e2e/utils.go

```diff
@@ -191,19 +191,12 @@ func createPVCAndApp(
 	return err
 }
 
-// createPVCAndDeploymentApp creates pvc and deployment, if name is not empty
-// same will be set as pvc and app name.
+// createPVCAndDeploymentApp creates pvc and deployment.
```

Comment:
Apologies for these not exactly relevant changes (in rbd.go as well), but they were asked for by some linter in `static-check`.

Reply:
its fine :)